.github/workflows/release.yml

```diff
@@ -51,12 +52,31 @@ jobs:
           elif [[ $BRANCH_NAME =~ ^feature/ ]]; then
```

Comment:
Need to update `release` and `feature` in to `major` and `minor` in the if/else

Reply:
Addressed